ckanext/fork/helpers.py

```diff
@@ -12,7 +17,7 @@ def get_parent_resource_details(resource_id):
         package = toolkit.get_action('package_show')(data_dict={'id': resource['package_id']})
     except toolkit.NotAuthorized:
         return {'success': False,
-                'msg': "Unable to access parent resource package information; current user may not be authorized to access this."}
+                'msg': "Unable to access parent resource information; current user may not be authorized to access this."}
```

Comment:
I'm not sure what you mean by "parent resource"?  Do you mean "parent dataset"?

Reply:
No, I mean the resource that has been forked - the upstream one. Like in Git where a branch has a parent branch sort of thing.

Reply:
The try catch above is about getting the parent "resource".  This one is then about getting the "parent resource's dataset/package". 

```suggestion
                'msg': "Unable to access parent resource's dataset; current user may not be authorized to access this."}
```

Reply:
Or, perhaps even better, I would write up the function like this:
```
def get_parent_resource_details(resource_id):
    try:
        resource = toolkit.get_action('resource_show')(data_dict={'id': resource_id})
        package = toolkit.get_action('package_show')(data_dict={'id': resource['package_id']})	
    except toolkit.NotAuthorized:
        return {'success': False,		 
                'msg': "Unable to access parent resource information; current user may not be authorized to access this."}		                
        }
    organization = package['organization']
    return {
        ...
    }
```

Reply:
There's no point in having them in separate try/except blocks, if you handle them in exactly the same way right?